mscore/pianoroll.cpp

```diff
@@ -512,13 +514,6 @@ void PianorollEditor::moveLocator(int i, const Pos& pos)
 void PianorollEditor::cmd(QAction* a)
       {
       score()->startCmd();
```

Comment:
ok, then what the point in calling this function at all :) and connect the actions then :)